### PR TITLE
Add absolute_import to each .py file in bokeh/bokeh

### LINKED ISSUE #1988: Use absolute_import throughout
Every bokeh module should do `from __future__ import absolute_import` at the top, right after the module docstring (first import)

### PR DESCRIPTION
**Pull request by Maggie-M**
issues: closes #1988

### CODE REVIEW

File: bokeh/charts/__init__.py [hunk 1]
```diff
@@ -15,7 +16,22 @@
...
+    '''
+
+    return vplot(*args, **kwargs)
+
+@deprecated("Bokeh 0.8.2", "bokeh.plotting.hplot function")
+def HBox(*args, **kwargs):
+    ''' Generate a layout that arranges several subplots horizontally.
+    '''
+
+    return hplot(*args, **kwargs)
```
Comment (mattpap) [inline bokeh/charts/__init__.py]: Why this is part of this PR?

File: bokeh/plotting.py [hunk 2]
```diff
@@ -14,12 +15,27 @@
...
+    return vplot(*args, **kwargs)
+
+@deprecated("Bokeh 0.8.2", "bokeh.plotting.hplot function")
+def HBox(*args, **kwargs):
+    ''' Generate a layout that arranges several subplots horizontally.
+    '''
+
+    return hplot(*args, **kwargs)
+
+
```
Comment (mattpap) [inline bokeh/plotting.py]: Same here.

Comment (Maggie-M) [inline bokeh/plotting.py]: Oh, whoops!  Thanks for catching those.  That was from another branch that I accidentally added into this one.  I'll take care of that right away.

File: bokeh/plotting.py [hunk 3]
```diff
@@ -14,7 +15,7 @@
 
 # extra imports -- just things to add to 'from plotting import *'
 from .document import Document
-from .models import ColumnDataSource, HBox, VBox
```
Comment (bryevdv) [inline bokeh/plotting.py]: I think the tests will pass if this line is restored as well.

Comment (bryevdv) [inline bokeh/plotting.py]: The the other PR can remove it when it adds the deprecated "function" versions

**Pull request merged**